### PR TITLE
a start on Transactions page. I just have a basic table.

### PR DESCRIPTION
**Pull request by asbel**

### CODE REVIEW

File: assets/stylesheets/transactions/transactions.css [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#transactions{
+	position: relative;
+	position: center;
+	background-color: #00000;
```
Comment (nickdemianchuk) [inline assets/stylesheets/transactions/transactions.css]: not even a color :)

Comment (asbel) [reply assets/stylesheets/transactions/transactions.css]: hahaha thanks
I still have a long way to go

File: app/transactions/transactions.html [hunk 2]
```diff
@@ -6,6 +6,7 @@
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <link rel="stylesheet" href="../../assets/stylesheets/shared/constants.css">
     <link rel="stylesheet" href="../../assets/stylesheets/shared/navbar.css">
+    <link rel="stylesheet" type="text/css" href="../../assets/stylesheets/transactions/transactions.css">
```
Comment (nickdemianchuk) [inline app/transactions/transactions.html]: either remove `type="text/css" ` or add it to each and every link for each of our pages to be consistent

File: app/transactions/transactions.html [hunk 3]
```diff
@@ -18,6 +19,66 @@
         <a href="#"><i class="fa fa-fw fa-user"></i> Log Out</a>
         <a href="#"><i class="fa fa-bell-o"></i> </a>
     </div>
+   
 </nav>
+ <div id="main">
```
Comment (nickdemianchuk) [inline app/transactions/transactions.html]: give the id a more meaningful name or rather use `main` element

File: app/transactions/transactions.html [hunk 4]
```diff
@@ -18,6 +19,66 @@
...
+    			</tr>
+    			<tr>
+    				<td><input type="checkbox" name="" value=""><br></td>
+    				<td>02/23/19</td>
+    				<td>Woodfield Mall</td>
+    				<td>Expense</td>
+    				<td>40</td>
+    				<td>Expense</td>
+    			</tr>
+
```
Comment (nickdemianchuk) [inline app/transactions/transactions.html]: remove blank lines here

File: assets/stylesheets/transactions/transactions.css [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+}
+#main{
```
Comment (nickdemianchuk) [inline assets/stylesheets/transactions/transactions.css]: start the styling from the outermost element
`main `goes first and then its child elements `table`, etc.

**Review by nickdemianchuk — CHANGES_REQUESTED** [review]
please address all of my comments

File: app/transactions/transactions.html (hunk 3 above)
Comment (asbel) [reply app/transactions/transactions.html]: switched to main element

File: assets/stylesheets/transactions/transactions.css [hunk 6]
```diff
@@ -1,11 +1,10 @@
-#transactions{
+main{
 	position: relative;
-	position: center;
-	background-color: #00000;
+	left : 50px;
 }
-#main{
+#transactions{
 	position: relative;
```
Comment (nickdemianchuk) [inline assets/stylesheets/transactions/transactions.css]: `position: relative` is redundant 
you overwrite it in the next line

File: app/transactions/transactions.html [hunk 7]
```diff
@@ -19,5 +19,62 @@
         <a href="#"><i class="fa fa-bell-o"></i> </a>
     </div>
 </nav>
+ <main>
+ 	<h2>Most recent transactions</h2>
+    	<table id="transactions">
+    		<thead>
+    			<tr>  
+	   				<th><input type="checkbox" name="" value=""><br></th>
```
Comment (nickdemianchuk) [inline app/transactions/transactions.html]: fix the alignment

File: app/transactions/transactions.html [hunk 8]
```diff
@@ -19,5 +19,62 @@
...
+    			<tr>
+    				<td><input type="checkbox" name="" value=""><br></td>
+    				<td>02/11/19</td>
+    				<td>Starbucks Coffee</td>
+    				<td>Coffee</td>
+    				<td>10.64</td>
+    				<td>Expense</td>
+    			</tr>
+    			<tr>
+	   				<td><input type="checkbox" name="" value=""><br></td>
```
Comment (nickdemianchuk) [inline app/transactions/transactions.html]: same

File: app/transactions/transactions.html [hunk 9]
```diff
@@ -19,5 +19,62 @@
...
+    				<td>02/19/19</td>
+    				<td>ParkChicago</td>
+    				<td>Parking</td>
+    				<td>3.49</td>
+    				<td>Expense</td>
+    			</tr>
+    			<tr>
+       				<td><input type="checkbox" name="" value=""><br></td>
+	   				<td>02/20/19</td>
```
Comment (nickdemianchuk) [inline app/transactions/transactions.html]: same

File: assets/stylesheets/transactions/transactions.css [hunk 10]
```diff
@@ -0,0 +1,20 @@
...
+	position: center;
+}
+h2{
+	text-align: center;
+}
+table{
+	position: 100%;
+}
+tr,td{
+	padding: 20px;
```
Comment (nickdemianchuk) [inline assets/stylesheets/transactions/transactions.css]: move it to constants

**Review by nickdemianchuk — CHANGES_REQUESTED** [review]
looks better but still needs to be changed ;)

**Review by nickdemianchuk — APPROVED** [review]

**Pull request merged**